File: SoalB/Java/CPU.java
public class CPU {
    /* atribut */
    private int priceCPU;
    private String nameCPU;

    /* constructor */
    public CPU(int priceCPU, String nameCPU) {
        this.nameCPU = nameCPU;
        this.priceCPU = priceCPU;
    }

    public CPU() {

    }

    /* membuat get set method harga CPU */
    public void setPriceCPU(int priceCPU) {
        this.priceCPU = priceCPU;
    }

    public int getPriceCPU() {
        return priceCPU;
    }

    /* membuat get set method nama CPU */
    public void setNameCPU(String nameCPU) {
        this.nameCPU = nameCPU;
    }

    public String getNameCPU() {
        return nameCPU;
    }

    /* menampilkan data CPU */
    public void printCPU() {
        System.out.println("######################################");
        System.out.println("CPU Price(IDR)      : " + this.getPriceCPU());
        System.out.println("CPU Name            : " + this.getNameCPU());
        System.out.println("######################################");
    }
}

File: SoalB/PHP/CPU.php
<?php

class CPU
{
    /* atribut */
    private $nameCPU, $priceCPU;

    /* constructor */
    function __construct($priceCPU, $nameCPU)
    {
        $this->priceCPU = $priceCPU;
        $this->nameCPU = $nameCPU;
    }

    /* Membuat get set method harga CPU */
    function setPriceCPU($priceCPU)
    {
        $this->priceCPU = $priceCPU;
    }
    function getPriceCPU()
    {
        return $this->priceCPU;
    }

    /* Membuat get set method nama CPU */
    function setNameCPU($nameCPU)
    {
        $this->nameCPU = $nameCPU;
    }
    function getNameCPU()
    {
        return $this->nameCPU;
    }

    /* menampilkan hasil data CPU */
    function printCPU()
    {
        echo "******************************" . "<br/>";
        echo "CPU Price(IDR)        : " . $this->getPriceCPU() . "<br/>";
        echo "CPU Name              : " . $this->getNameCPU() . "<br/>";
        echo "******************************" . "<br/>";
    }

    /*destructor */
    function __destruct()
    {
    }
}
